specification/2.0/appendix/performance-metrics/README.md

```diff
@@ -0,0 +1,77 @@
+# Runtime Performance Metrics 
+
+The goal of these metrics is to asses the runtime performance needs of glTF assets.  
```

Comment:
Nitpick: it's "to assess", not "to asses". Here, the second "s" is more important than in many other cases.

Reply:
Lol - thanks for catching this :-)